### ISSUE TITLE
TTOU doesn't log anything

### ISSUE THREAD

**Issue by nateberkopec** [puma/puma#1147]
Looks confusing, even though when checking with `ps` it appears to be working correctly, killing the last spawned worker. Leads to output like:

```
[23860] Puma starting in cluster mode...
[23860] * Version 3.6.0 (ruby 2.3.2-p217), codename: Sleepy Sunday Serenity
[23860] * Min threads: 1, max threads: 1
[23860] * Environment: development
[23860] * Process workers: 2
[23860] * Phased restart available
[23860] * Listening on tcp://0.0.0.0:3000
[23860] Use Ctrl-C to stop
[23860] * Starting control server on unix:///var/folders/11/xdcy4xcj5vn8b202bzp1pym80000gn/T/puma-status-1479669005626-23860
[23863] + Gemfile in context: /Users/nateberkopec/Code/pumatest/Gemfile
[23864] + Gemfile in context: /Users/nateberkopec/Code/pumatest/Gemfile
[23860] - Worker 0 (pid: 23863) booted, phase: 0
[23860] - Worker 1 (pid: 23864) booted, phase: 0
[23871] + Gemfile in context: /Users/nateberkopec/Code/pumatest/Gemfile
[23860] - Worker 2 (pid: 23871) booted, phase: 0
[23893] + Gemfile in context: /Users/nateberkopec/Code/pumatest/Gemfile
[23860] - Worker 2 (pid: 23893) booted, phase: 0
```

In the above output I sent TTIN, TTOU, then TTIN again.

See #1103

**Comment by dannyfallon** [comment]
This was fixed in #1165 and I think it can be closed. Here's the output in my local env running Puma and sending TTOU followed by 2xTTIN:

```
[4464] - Worker 1 (pid: 4470) booted, phase: 0
[4464] - Worker 2 (pid: 4474) booted, phase: 0
[4464] - Worker 3 (pid: 4478) booted, phase: 0
[4464] - Worker 0 (pid: 4466) booted, phase: 0
>> Send TTOU
[4464] - Worker 3 (pid: 4478) terminating
>> Send TTIN (twice)
[4464] - Worker 3 (pid: 4577) booted, phase: 0
[4464] - Worker 4 (pid: 4583) booted, phase: 0
```

**Comment by nateberkopec** [comment]
Tada!
